packages/proxy/src/proxy.ts

```diff
@@ -65,6 +71,12 @@ export interface CacheKeyOptions {
   excludeOrgName?: boolean;
 }
 
+export interface SpanLogging {
```

Comment:
nit: for uniformity, should we name this type `SpanLogger`?

Reply:
yep